sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -10,13 +10,6 @@
     - `VOIP_JOIN` (Access to Calling APIs but without the authorization to start new calls)
 - Added a new API version `ApiVersion.V2023_10_01` that is now the default API version.
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
-- The `MicrosoftBotIdentifier` and `MicrosoftBotProperties` have been deprecated.
```

Comment:
We could probably leave this in the changelog, as it has been removed since the last beta, though maybe just state that it's been "removed" rather than "deprecated"?

Reply:
Updated